x-pack/plugin/core/template-resources/src/main/resources/ml/meta_index_mappings.json

```diff
@@ -36,6 +36,15 @@
       "end_time": {
         "type": "date"
       },
+      "skip_result": {
+        "type": "keyword"
+      },
+      "skip_model_update": {
+        "type": "keyword"
+      },
+      "force_time_shift": {
+        "type": "keyword"
```

Comment:
ES has a `boolean` [mapping type](https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-types.html), can that be used instead of keyword for `skip_result` and `skip_model_update` 

`force_time_shift` can be `type: integer`

Reply:
I'll update this.